runtime/tests/port/j9vmemTest.c

```diff
@@ -2238,14 +2260,43 @@ j9vmem_testFindValidPageSize_impl(struct J9PortLibrary *portLibrary, char *testN
 		expectedPageFlags = defaultLargePageFlags;
 		expectedIsSizeSupported = FALSE;
 	} else {
-		expectedPageSize = FOUR_KB;
+		expectedPageSize = defaultPageSize;
 		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
 		expectedIsSizeSupported = FALSE;
 	}
 
 	verifyFindValidPageSizeOutput(portLibrary, testName,
-								expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
-								requestedPageSize, requestedPageFlags, isSizeSupported);
+	                              expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
+	                              requestedPageSize, requestedPageFlags, isSizeSupported);
+
+	/* 2M is only possible for Linux */
```

Comment:
Missing period, suggest "on" is better than "for" in reference to a platform,
```c
	/* 2M is only possible on Linux. */
```
The comment says "Linux", but the condition is more specific: `LINUXPPC`.